sljit_src/sljitLir.h

```diff
@@ -1774,6 +1774,7 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_fmem_update(struct sljit_compiler
      - the memory operation (op) and the base address (stored in mem_reg)
        passed to the load/store operations must be the same (the mem_reg
        can be a different register, only its value must be the same)
+     - an store must always follow a load for the same transaction.
```

Comment:
I hope we can avoid such requirement. Sometimes you want to do conditional updates, and want to do nothing if the condition failed.

Is there a cpu where this is required?

Reply:
> Sometimes you want to do conditional updates, and want to do nothing if the condition failed.

Could you elaborate?

> Is there a cpu where this is required?

Presume all CPUS that will be implemented using LL/SC; definitely ARM64 will always fail the `stxr` call, unless the memory address that it refers to was first added to the monitor by its previous `lxdr`.

Reply:
I am sorry. I misread the text.